docs/get-started/bridge/shimmerbridge.md

```diff
@@ -0,0 +1,230 @@
+
+## Bridge Assets with ShimmerBridge
+
+**1.)** Enter **https://shimemrbridge.org/terms** and read the terms and conditions carefully before you start using ShimmerBridge. Once you understand the terms and conditions, enter the ShimmerBridge User Interface and connect your Web3 Wallet.
+
+
+![Layer Zero UI1](/img/introduction/Layer Zero UI1.png)
+
+**2.)** **Select the source chain** - from the source chain you want to bridge a token into Shimmer EVM. To send tokens out of Shimmer EVM, select Shimmer EVM as the source chain.
+
+
+**3.)** Click "**Select Token**" - choose a token to transfer from the source chain. The Interface will display all bridgeable tokens and your token balance in the selection dropdown. Also, all other bridgeable tokens you hold in one of the other available networks are displayed. 
+- The app will automatically select Shimmer EVM as your destination chain in case you choose another network than Shimmer EVM. The app will always set the matching token on the destination chain based on your source chain token selection.
+
+**4.)** **Enter the number of tokens you want to bridge from the source chain**
+- The app will display:
+    -  the amount of bridged tokens you will receive on the destination chain. 
+    - The necessary fees for the token transfer are displayed in the native gas token of the source chain. Ensure you have enough gas tokens on the source chain to pay the fees.
+
+::: info
+
+**Gas on Destination:**
+If you do not own any gas token on the destination chain (in this case, SMR on Shimmer EVM), you can arrange an airdrop of native gas tokens to your wallet as part of your transfer. This will increase your fee but enable you to have the necessary amount of gas in the destination chain for further activities.
+Either use "Max," which gives you 0.25 SMR, or specify a custom amount.
+
+::::
+
+**5.)** **Transfer the tokens**
+- Click "**Transfer**". Your wallet will ask you to approve the token spending and confirm the transaction
+- After your confirmation, a popup will occur; if you click the link, you are forwarded to [LayerZero Scan](https://layerzeroscan.com/), to observe the detailed progress of your bridge interaction.
+- The ShimmerBridge app will also display the progress of your transaction, including an estimated period till the bridging is completed in the "transactions" box that appears after confirmation.
+- Once the status changes to "completed", your tokens have arrived in your wallet.
+- Ensure you have imported the correct token into your wallet on the destination chain to see it.
+
+## Troubleshooting
+
+Conditions that will hinder your transfer will be indicated in the "Action button" as follows:
+
+**`Insufficient Balance`**
+
+- You entered a higher amount of tokens than you hold in your account.
+    -  Reduce the amount you want to transfer or add more tokens to your wallet.
+
+
+**`Not enough native for Gas`**
```

Comment:
```suggestion

### Not Enough Native for Gas
```

Reply:
shouldn't the error message be "Not Enough Native Tokens for Gas"?

Reply:
The message is currently "Not enough Native for Gas" - we will look if we can change that and include tokens